shared/liboxide/oxide_sentry.cpp

```
@@ -30,6 +30,26 @@
 #include <cpptrace/cpptrace.hpp>
 #include <cpptrace/from_current.hpp>
 #include <iostream>
+#include <signal.h>
+
+void sigsegv_handler(int signo, siginfo_t* info, void* context){
+    Q_UNUSED(signo);
+    Q_UNUSED(info);
+    Q_UNUSED(context);
+    const char* message = "Segmentation fault\n";
+    write(STDERR_FILENO, message, strlen(message));
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)